NeXus/NXxpcs.nxdl.xml

```diff
@@ -76,146 +76,48 @@
-                  @units = 1/angstrom ->
-
-      -->
-      <doc>Additions to the NXcanSAS definition.</doc>
-      <group type="NXdata" name="data">
-        <field name="I_partial" type="NX_NUMBER">
-          <doc>
-            I(Q) for stability plot.
+      <field name="frame_sum" type="NX_NUMBER" units="NX_ANY">
+        <doc>average intensity v. time</doc>
+      </field>
+      <field name="g2" type="NX_NUMBER" units="NX_ANY">
+        <doc>TODO:</doc>
+      </field>
+      <field name="g2_stderr" type="NX_NUMBER" units="NX_ANY">
+        <doc>TODO:</doc>
+      </field>
+      <field name="tau" type="NX_INT" units="NX_ANY">
+        <doc>
+        delay time, multitau
 
-            Q: Should units match I?
-          </doc>
-        </field>
-        <!-- defined in NXcanSAS -->
-        <field name="I" type="NX_NUMBER" />
-        <field name="Q" type="NX_NUMBER" units="NX_PER_LENGTH" />
-      </group>
+        t_el, this is integer delta frame units, not time elapsed units
+        </doc>
+      </field>
     </group>
-
-    <group type="NXcanSAS" name="SAXS_2D">
-      <!--
-        https://manual.nexusformat.org/classes/applications/NXcanSAS.html
-
-        TODO: Should this be type="NXsubentry" with definition field of NXcanSAS?
-
-          SAXS_2D:NXsubentry
-            @NX_class = NXsubentry
-            @canSAS_class = SASentry
-            @default = data
-            definition:NX_CHAR = NXcanSAS
-            instrument -> /entry/instrument
-            run -> /entry/title
-            sample -> /entry/sample
-            title:NX_CHAR = static 2D SAXS from XPCS data
-            data:NXdata
-              @NX_class = NXdata
-              @canSAS_class = SASdata
-              @mask = mask
-              @signal = I
-              I:NX_FLOAT32[516,1556] = __array
-                __array = [
-                    [0.0029529633, 0.0049402798, 0.006318577, '...', 0.0109847365]
-                    [0.0043127504, 0.004439526, 0.005501271, '...', 0.004218236]
-                    [0.008683259, 0.0066289976, 0.003876593, '...', 0.0056484076]
-                    ...
-                    [0.004690421, 0.0072740484, 0.0053803907, '...', 0.0039639533]
-                  ]
-                @units = arbitrary
-                @units_details = Photon/Pixel/Frame
-              mask:NX_INT64[516,1556] = __array
-                __array = [
-                    [0, 0, 0, '...', 0]
-                    [0, 0, 0, '...', 0]
-                    [0, 0, 0, '...', 0]
-                    ...
-                    [0, 0, 0, '...', 0]
-                  ]
-                @units = boolean
-
-      -->
-      <doc>Additions to the NXcanSAS definition.</doc>
-      <!-- TODO: any?-->
-      <!-- defined in NXcanSAS -->
-      <group type="NXdata" name="data">
-        <field name="I" type="NX_NUMBER" />
-        <field name="Q" type="NX_NUMBER" units="NX_PER_LENGTH" />
-      </group>
+    <group type="NXdata" name="twotime">
```

Comment:
so here the name of NXdata is "twotime" but why not "xpcs" in 79?

Reply:
Sounds good.  Let's make the change.

Reply:
done